### PR TITLE
Update maps to point to new radar logo

### PR DESCRIPTION
**Pull request by tjulien**
Here's a screenshot pointing to the new logo on staging:

https://documentation-git-add-map-logo-to-assets-radarlabs.vercel.app/documentation/maps/maps

Screenshot 2023-05-19 at 10 17 07 AM

### CODE REVIEW

File: docs/maps/maps.mdx [hunk 1]
```diff
@@ -72,7 +72,7 @@ To create a simple web page with a map:
...
+        <img alt="Radar" src="https://api.radar.io/maps/images/logo.svg" height="38" width="80" />
```
Comment (kochis) [inline docs/maps/maps.mdx:75]: This will be `/maps/static/images/logo.svg`

**Comment by tjulien** [comment]
@kochis should be ready for another look

**Comment by jsani-radar** [comment]
hmm just looked at the link @tjulien posted above and it seems to be missing?

Screenshot 2023-05-19 at 11 54 45 AM

**Comment by tjulien** [comment]
@kochis I changed it back to prod, the screenshot in the PR description is what staging looks like

**Comment by jsani-radar** [comment]
yup all good, my misunderstanding

**Pull request merged**